src/main/assembly/fakeroot.xml

```diff
@@ -103,6 +103,12 @@ Installed in a fakeroot directory structure.
             <outputDirectory>/etc/init.d</outputDirectory>
             <fileMode>0755</fileMode>
         </fileSet>
+	<!-- .info files -->
+        <fileSet>
+            <directory>src/main/infofiles</directory>
+            <outputDirectory>/etc/grid-security/certificates</outputDirectory>
```

Comment:
> The point is that the PIP looks in the 'trustDir' (i.e. normally /etc/grid-security/certificates) for all the info file

The point is: are these files meant to be used by other services? If not, they should be packaged in an argus-owned directory, and the PIP modified to look into this directory.
I think the fact that the policy files and certificates must live in the same dir is only a limitation of the current PIP policy lookup implementation, or there's more to it?